packaging/flatpak/com.freerdp.FreeRDP.json

```diff
@@ -74,53 +86,80 @@
             {
                 "type": "git",
                 "url": "https://github.com/LudovicRousseau/PCSC.git",
-                "tag": "1.9.5"
+                "tag": "1.9.9"
             }
+        ],
+				"cleanup": [
+					"/share/doc",
+					"/share/man"
+				]
+    }, 
+		{
+			"name": "openssl111",
+			"buildsystem": "simple", 
+			"build-commands": [
+				"./config --prefix=/app",
+				"make -j $FLATPAK_BUILDER_N_JOBS",
+				"make install"
+			],
+			"cleanup": [
+				"/share/doc",
+				"/share/man",
+				"/bin"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.openssl.org/source/openssl-1.1.1q.tar.gz",
+					"sha256": "d7939ce614029cdff0b6c20f0e2e5703158a489a72b2507b8bd51bf8c8fd10ca"
+				}
+			]
+		},
+    {
+        "name": "freerdp",
+        "buildsystem": "cmake-ninja",
+        "cleanup": [
+        ],
+        "config-opts": [
+					"-DCMAKE_VERBOSE_MAKEFILE=ON", 
+					"-DCMAKE_BUILD_TYPE:STRING=Release",
+					"-DCMAKE_INSTALL_LIBDIR:PATH=lib",
+					"-DCHANNEL_TSMF:BOOL=OFF",
+					"-DCHANNEL_URBDRC:BOOL=ON",
+					"-DBUILD_TESTING:BOOL=OFF",
+					"-DWITH_MANPAGES:BOOL=OFF",
+					"-DWITH_GSSAPI:BOOL=OFF",
+					"-DWITH_PCSC:BOOL=ON",
```

Comment:
`-DWITH_PKCS11:ON` ?

Reply:
@hardening need to check if that is available from runtime.